packages/upload/src/FzUpload.vue

```diff
@@ -68,12 +68,18 @@ const props = withDefaults(defineProps<FzUploadProps>(), {
   buttonLabel: "Carica",
   dragAndDropLabel: "o trascina qui",
 });
-const emit = defineEmits([]);
+const emit = defineEmits(["fzupload:change"]);
 const model = defineModel<File[]>();
 
 const input = ref<HTMLInputElement | null>(null);
 const urlByFileMap = ref(new Map<File, string>());
 
+onMounted(() => {
+  if(model.value && model.value.length > 0 && !props.multiple) {
+    console.warn("[Fiscozen Design System]: FzUpload prop 'multiple' is set to false, but multiple files are provided. Only the first file will be used.");
+  }
```

Comment:
I don't see any logic that stops me from passing more than one file to the model